test-metadata-generation.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+source tools.sh
+source createdisk-library.sh
+source snc-library.sh
+
+# If the user set OKD_VERSION in the environment, then use it to override OPENSHIFT_VERSION, set BASE_OS, and set USE_LUKS
+# Unless, those variables are explicitly set as well.
+OKD_VERSION=${OKD_VERSION:-none}
+if [[ ${OKD_VERSION} != "none" ]]
+then
+    OPENSHIFT_VERSION=${OKD_VERSION}
+    BASE_OS=fedora-coreos
+fi
+BASE_OS=${BASE_OS:-rhcos}
+USE_LUKS=${USE_LUKS:-false}
```

Comment:
We can remove the `USE_LUKS` variable now since it is now false for RHCOS and also for FCOS.

Reply:
Ah I'll look at adding an additional patch for this.